backend/index.js

```diff
@@ -0,0 +1,44 @@
+const express    = require('express');
+const mongoose   = require('mongoose');
+const helmet     = require('helmet');
+const bodyParser = require('body-parser');
+const morgan     = require('morgan');
+const bluebird   = require('bluebird');
+const path = require('path');
+const expresshandlebars = require('express-handlebars');
+
+const config = require('./config');
+const routes = require('./routes');
+
+const app = express();
+
+mongoose.Promise = bluebird;
+mongoose.connect(config.mongo.url);
+
+app.use(helmet());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(morgan('tiny'));
+
+// Specify where our static assets are located
+app.use(express.static(path.join(__dirname, '../../client/public')));
```

Comment:
Change the path to the public route from `../../client/public` to `../client/public`

Reply:
done